modules/osint/bing_mobile_view.py

```diff
@@ -71,7 +71,7 @@ def module_api(self):
 
 	run = self.bing_mobile_view(url)
 
-	if not run.screenshot(): # If some error has occured while taking screenshot, return output to avoid further errors.
```

Comment:
Here, when screenshot is successful, it return a None type. As a result, this evaluates to True and the module exits with empty output. So used if run,screenshot()==False.